doc/en/goodpractises.rst

```diff
@@ -4,29 +4,30 @@
 Good Integration Practices
 =================================================
 
-Work with virtual environments
------------------------------------------------------------
 
-We recommend to use virtualenv_ environments and use pip_
-(or easy_install_) for installing your application and any dependencies
-as well as the ``pytest`` package itself.  This way you will get an isolated
-and reproducible environment.    Given you have installed virtualenv_
-and execute it from the command line, here is an example session for unix
-or windows::
+.. _`test discovery`:
+.. _`Python test discovery`:
 
-    virtualenv .   # create a virtualenv directory in the current directory
+Conventions for Python test discovery
+-------------------------------------------------
 
-    source bin/activate  # on unix
+``pytest`` implements the following standard test discovery:
 
-    scripts/activate     # on Windows
+* collection starts from paths specified in :confval:`testpaths` if configured,
+  otherwise from initial command line arguments which may be directories,
```

Comment:
that's ambiguous.  https://pytest.org/latest/customize.html#confval-testpaths is more precise in saying that if you are specifying command line args "testpaths" is not considered.